src/soundio/soundmanagerconfig.cpp

```diff
@@ -10,8 +10,10 @@
 
 const QString SoundManagerConfig::kDefaultAPI = QStringLiteral("None");
 const QString SoundManagerConfig::kEmptyComboBox = QStringLiteral("---");
-// Sample Rate even the cheap sound Devices will support most likely
-const unsigned int SoundManagerConfig::kFallbackSampleRate = 48000;
+constexpr mixxx::audio::SampleRate SoundManagerConfig::kMixxxDefaultSampleRate =
+        mixxx::audio::SampleRate(44100);
+constexpr mixxx::audio::SampleRate SoundManagerConfig::kFallbackSampleRate =
+        mixxx::audio::SampleRate(48000);
```

Comment:
```suggestion
const mixxx::audio::SampleRate SoundManagerConfig::kMixxxDefaultSampleRate =
        mixxx::audio::SampleRate(44100);
const mixxx::audio::SampleRate SoundManagerConfig::kFallbackSampleRate =
        mixxx::audio::SampleRate(48000);
```
This produces a linker error with ldd.
It should be anyway better to match the definition in the header.

Reply:
make both constexpr, move to header instead? Or would you consider that to leak implementation details?

Reply:
https://github.com/mixxxdj/mixxx/pull/11904/commits/987fd3a461a8133cd073e5285b157f524a025ca3 ok?